services/api/src/utils/middleware/error-handler.js

```diff
@@ -1,9 +1,19 @@
+const config = require('@bedrockio/config');
+
+const ENV_NAME = config.get('ENV_NAME');
+
 async function errorHandler(ctx, next) {
   try {
     await next();
   } catch (err) {
-    let { status = 500, message, details } = err;
+    let { status, message, details } = err;
 
+    if (!status) {
+      status = 500;
+      if (ENV_NAME === 'production') {
```

Comment:
I would use ENV_NODE to include staging in there too. So both staging and production behaves the same way.

Reply:
hmmmm that's fine and I agree in most cases but this is masking error messages that we MIGHT want to catch in staging before going to production... this might be one of the few "staging should match production" situations

Reply:
i mean its still caught, e.g. appears in the logs the correct error + if you get 500 from the api regardless of message you should look into that.

Reply:
Right but it then you have to open up the logs and hunt it down as opposed to getting a hint as instant feedback...
imho it increases FRICTION (using Dom's trigger word on purpose here ;) )